src/components/TagGeneratorInstructions.js

```diff
@@ -116,6 +116,11 @@ const BottomSection = () =>{
 
 const HowToUse = () => {
   const classes = useStyles()
+  const breadCrumbLinks = [
+    { name: 'Home', href: '/home' },
+    { name: 'Join the Index', href: '/join-index' },
+    { name: 'How to Add Your Project', href: '/join-index/how-to-add' },
```

Comment:
The updated breadcrumbs info is in https://docs.google.com/spreadsheets/d/163Pqh-TPfxZySPjZvje4VKouaNSUxXFuDFLVQ9VW0UQ/edit#gid=0 . The breadcrumb should look like this：‘Home / How to Add your Project’. Please remove line 121